src/plugins/condition/StyleRuleManager.js

```diff
@@ -103,6 +126,21 @@ export default class StyleRuleManager extends EventEmitter {
         }
     }
 
+    applySelectedConditionStyle() {
+        const conditionId = this.selectedConditionId || this.defaultConditionId;
+        if (conditionId) {
+            let foundStyle = this.conditionalStyleMap[this.selectedConditionId];
```

Comment:
wondering in code `let foundStyle = this.conditionalStyleMap[this.selectedConditionId];`
can we use `conditionId ` instead of `this.selectedConditionId` ?

Reply:
Yes, that's a good catch.